salt/top.sls

```diff
@@ -4,6 +4,7 @@ base:
     - ca
   'roles:(admin|kube-(master|minion))':
     - match: grain_pcre
+    - ca-cert
```

Comment:
The `ca-cert` is required in the admin because as part of the orchestration we try to connect to dex using the ca file present in `/etc/pki/trust/...` as in the rest of the machines. I won't harm explicitly making the admin include `ca-cert` too.